core/src/test/java/jenkins/util/ResourceBundleUtilTest.java

```diff
@@ -54,6 +68,9 @@ public void test_known_locale() {
      */
     @Test
     public void test_unknown_locale() {
+        //Set Default-Locale to english
+        Locale.setDefault(new Locale("en", "US"));
+
```

Comment:
Why not make it more local? I mean, remove the @BeforeClass and @AfterClass and put the lines just here before and after. IIRC there's no other tests concerned right?